_LowLevelCode/cpp/test/combine_sqw.tests/combine_sqw.test.cpp

```diff
@@ -706,10 +706,10 @@ TEST_F(TestCombineSQW, SQW_Reader_Read_All) {
     std::cout << "\n Time to run single thread with 1024 words buffer: " << t_end << "ms\n";
 
 
-    for (size_t i = 0; i < pix_buffer.size(); i++) {
-        ASSERT_EQ(pix_buffer[i], pixels[i]);
-        pix_buffer[i] = 0;
-    }
+    // for (size_t i = 0; i < pix_buffer.size(); i++) {
```

Comment:
For a lot of these tests since all the `ASSERT_EQ` following the setup code block are disabled there is no need to execute the setup code either and that should be commented too.

Refactoring the test case into multiple test functions should be performed as part of the 'fix' to these failing tests.